prow/plank/controller.go

```diff
@@ -208,29 +208,30 @@ func (c *Controller) Sync() error {
 // TODO: Dry this out - need to ensure we can abstract children cancellation first.
 func (c *Controller) terminateDupes(pjs []kube.ProwJob) error {
 	// "job org/repo#number" -> newest job
-	dupes := make(map[string]kube.ProwJob)
+	dupes := make(map[string]int)
 	for i, pj := range pjs {
 		if pj.Complete() || pj.Spec.Type != kube.PresubmitJob {
 			continue
 		}
 		n := fmt.Sprintf("%s %s/%s#%d", pj.Spec.Job, pj.Spec.Refs.Org, pj.Spec.Refs.Repo, pj.Spec.Refs.Pulls[0].Number)
 		prev, ok := dupes[n]
 		if !ok {
-			dupes[n] = pj
+			dupes[n] = i
 			continue
 		}
-		toCancel := pj
-		if prev.Status.StartTime.Before(pj.Status.StartTime) {
-			toCancel = prev
-			dupes[n] = pj
+		cancelIndex := i
+		if pjs[prev].Status.StartTime.Before(pj.Status.StartTime) {
+			cancelIndex = prev
+			dupes[n] = i
 		}
+		toCancel := pjs[cancelIndex]
 		toCancel.Status.CompletionTime = time.Now()
 		toCancel.Status.State = kube.AbortedState
-		npj, err := c.kc.ReplaceProwJob(toCancel.Metadata.Name, toCancel)
+		pjutil, err := c.kc.ReplaceProwJob(toCancel.Metadata.Name, toCancel)
```

Comment:
rename this in the jenkins side to npj instead of renaming this to pjutil.

Reply:
done.